.ruby-version

```diff
@@ -1 +1 @@
-3.1.1
+2.6.6
```

Comment:
Isn't there any combination of dependencies that keep the Ruby version at or above 3.1.1? This was last bumped 2 years ago and it sounds like a regression to revert back to a [unsupported](https://endoflife.date/ruby) Ruby version 😬😬

Reply:
I know, but that's what works fastlane/fastlane — the CircleCI and images use that already. It seems that Netlify can't work with non-default bundler otherwise so downgrading Netlify to the same env as CircleCI, not introducing nothing "new", see my elaboration: https://github.com/fastlane/docs/issues/1234#issuecomment-1962724630

Reply:
Got it 😅 

Hey can you DM me on https://twitter.com/rogerluan_ or email me? (Email is in my GH profile) — I wanna reach out to you about something but couldn't find a contact :)!

Reply:
@rogerluan That's intentional;) Look around your filtered folders for my username, or just gmail me at that…

Reply:
BTW the .ruby-version is not really needed for CircleCI or local work, quite the contrary, so if someone feels fancy enough to just print the file during a CI run (specifically, a PR targeted preview CI only, not elsewhere), after checkout, before configuring the runtime env, this could be better fixed somewhere in the tooling to shim it only for Netlify previews. But I'm not a hotshot like that so it's just an idea for someone more comfortable with breaking and fixing things properly;D

This sucks because it basically ask for a zombie version for local work (pretty much a macOS default Ruby version FWIW;D) even thou the docs project builds happily at more current 3.x (and that should be used in local dev anyways…), I just don't have the imagination to fix the two disjoining pipelines without messing up the local workspace TBH:/

Reply:
Just for the laughs — if you omit the ruby version completely, Netlify running the same image (note the sha) will pick different ruby versions for fastlane/docs ❌ and janbrasna/docs ✅: https://github.com/fastlane/docs/pull/1244#issuecomment-2040449320 🤷

_#givinup_

Reply:
So to match what's now running in the test docker, I've tried the highest security update version for Ruby 2.x ever — the 2.7.8 — and that seems to work in both workflows. Hopefully that's the least lame we can do now.